doc/example/reference.cpp

```diff
@@ -490,14 +497,17 @@ namespace invocation_example_4
 //[ invocation_example_4
 MOCK_CLASS( mock_class )
 {
-    MOCK_STATIC_METHOD( method, 1, void( int ) )
+    MOCK_STATIC_METHOD( method1, 1, void( int ) )
+    MOCK_STATIC_METHOD( method2, 1, void( int ) )
```

Comment:
Why two examples for this one in particular?

Reply:
Because each gets one assertion added. I found that cleaner but could probably also add both to one function